packages/edit-navigation/src/components/menus-editor/index.js

```diff
@@ -19,33 +25,65 @@ import CreateMenuArea from './create-menu-area';
 import NavigationEditor from '../navigation-editor';
 
 export default function MenusEditor( { blockEditorSettings } ) {
-	const { menus, hasLoadedMenus } = useSelect( ( select ) => {
-		const { getMenus, hasFinishedResolution } = select( 'core' );
-		const query = { per_page: -1 };
-		return {
-			menus: getMenus( query ),
-			hasLoadedMenus: hasFinishedResolution( 'getMenus', [ query ] ),
-		};
-	}, [] );
-
+	const [ menuId, setMenuId ] = useState();
+	const [ showCreateMenuPanel, setShowCreateMenuPanel ] = useState( false );
 	const [ hasCompletedFirstLoad, setHasCompletedFirstLoad ] = useState(
 		false
 	);
 
+	const noticeId = useRef();
+
+	noticeId.current = uniqueId(
```

Comment:
This will create a new ID on every render. And you shouldn't mutate refs outside effects.